dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -756,6 +756,8 @@ info.resumeFailedException=Resuming the {0} server failed. Check the server logs
 info.resumingListeners.target=Resuming the specified target components of the {0} server.
 info.resumedListeners.target=Resuming the specified target components of the {0} server completed.
 info.resumeFailedException.target=Resuming the specified target components of the {0} server failed. Check the server logs for details.
+info.serverCommandAuthFailure=The command {0} failed because the process could not delete the file {1}.
```

Comment:
Change
"The command {0} failed because the process **could** not delete the file {1}."
to
"The command {0} failed because the process **might** not delete the file {1}."

Reply:
Hi, 
I'm not sure that change makes sense here. If this error message is issued, it is definite that the file was not able to be deleted. 'Might' would imply that it may or may not have been able to delete the file. It would also be inconsistent with other failures from the same script, all of which use 'could not' when something was not able to be completed.

Reply:
Hi @brenthdaniel. I understand, and thanks for your feedback. Okay to leave it as is.